scripts/api-inference/scripts/generate.ts

```diff
@@ -99,10 +99,36 @@ const TASKS_DATA = (await response.json()) as any;
 //// Snippet utils ////
 ///////////////////////
 
+const formatSnippets = (result: snippets.types.InferenceSnippet | snippets.types.InferenceSnippet[], defaultClient: string, language: string): string => {
+  // For single snippet, just wrap with code block
+  if (!Array.isArray(result) || result.length === 1) {
+    const snippet = Array.isArray(result) ? result[0] : result;
+    return `\`\`\`${language}\n${snippet.content}\n\`\`\``;
+  }
+  
+  // For multiple snippets, add description and wrap each one
+  return result
+    .map(snippet => {
+      const client = snippet.client || defaultClient;
+      return `With ${client} client:\n\`\`\`${language}\n${snippet.content}\n\`\`\``;
```

Comment:
```suggestion
      return `Using `${client}`:\n\`\`\`${language}\n${snippet.content}\n\`\`\``;
```

(nit) slight preference for
```
Using `huggingface_hub`:
```
over 
```
With huggingface_hub client:
```

(especially because `requests` is not really a client per-se)

Reply:
fixed in [9e47139](https://github.com/huggingface/hub-docs/pull/1490/commits/9e471393d3c343fca6068d0f15f3934cc4e95990), i did not push the regenerated docs,  leaving that for the automated PR.